### PR TITLE
Docker and NKUST version

### PR DESCRIPTION
**Pull request by abc873693**
Update:
 README.MD for docker install
 python request redis host config by .env file
 nkust version for remove login limit  in **feature login**
Add:
 meassge and status if user info is empty in **feature user info**
Fix:
 flask deprecated library 
 **limit flask version by less than 1.0**

### CODE REVIEW

**Comment by abc873693** [comment]
@louisom

**Comment by abc873693** [comment]
@mlouielu

**Comment by abc873693** [comment]
@vn7n24fzkq  need you

File: src/kuas_api/kuas/leave.py [hunk 1]
```diff
@@ -50,20 +62,20 @@ def login(session, username, password):
...
     root = etree.HTML(
-        session.get("https://leave.kuas.edu.tw:446/AK002MainM.aspx").text)
+        session.get("http://leave.nkust.edu.tw/AK002MainM.aspx").text)
 
     form = {}
     for i in root.xpath("//input"):
         form[i.attrib["name"]] = i.attrib[
             "value"] if "value" in i.attrib else ""
-
+    #print(form)
```
Comment (vn7n24fzkq) [inline src/kuas_api/kuas/leave.py:71]: Avoid unused comment.
Same problem at line 221 to 224.

Comment (abc873693) [reply src/kuas_api/kuas/leave.py:71]: @takidog you see see

File: README.md [hunk 2]
```diff
@@ -1,25 +1,83 @@
...
+$ pip install -r requirements.txt
 $ redis-server &
 $ python src/web-server.py
 ```
+By docker  
+---
+Requirement
+* Redis instance running on localhost
+
+Need to add environment variable **REDIS_URL=redis://127.0.0.1:6379/0**
```
Comment (vn7n24fzkq) [inline README.md]: I think command with block will more easy to set env variable.
Like this.
```
$ export REDIS_URL=redis://127.0.0.1:6379/0
```
@abc873693

File: src/kuas_api/kuas/leave.py (hunk 1 above)
Comment (takidog) [reply src/kuas_api/kuas/leave.py:71]: I got it.
line 71 forget delete it :D

File: README.md (hunk 2 above)
Comment (abc873693) [reply README.md]: Fixed

**Pull request merged**